### PR TITLE
Enable secrets encryption by default in test clusters

### PR DESCRIPTION
**Pull request by linki**
Enable secrets encryption by default in test clusters.

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
We can't roll it out in one go, since API servers that have not been updated won't be able to read the secrets. Let's split into two.

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -289,11 +289,19 @@ enable_hpa_scale_to_zero: "true"
 
 # setup supporting components to enable encryption
 # this flag must only be switched from true to false when enable_encryption is false and all secrets were decrypted
+{{if eq .Environment "test"}}
+support_encryption: "true"
```
Comment (aermakov-zalando) [inline cluster/config-defaults.yaml:293]: I'd also rename the config item to something more specific (encryption of what?).

**Comment by linki** [comment]
I'm currently updating them. This is in preparation to drop the cluster-registry overrides in the future and to use encryption from the start for new clusters.

Do you still see issues with it?

**Comment by aermakov-zalando** [comment]
If they're manually updated then it should be fine.

File: cluster/config-defaults.yaml (hunk 1 above)
Comment (linki) [reply cluster/config-defaults.yaml:293]: I would defer this. The config item hopefully won't be here for long.

**Comment by linki** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**